### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by kenshaw**
Updating to latest chromedp and other dependencies.

### CODE REVIEW

**Comment by kenshaw** [comment]
Recently used this for doing some tests in a browser (not related to `chromedp`). Updated the deps to keep it in sync with the other dependencies.

File: go.mod [hunk 1]
```diff
@@ -1,12 +1,20 @@
 module github.com/agnivade/wasmbrowsertest
 
-go 1.12
+go 1.17
```
Comment (agnivade) [inline go.mod:3]: Can we keep this to 1.12?

**Comment by kenshaw** [comment]
@agnivade I've rolled back the Go version and pushed the commit. Thanks for the tool!

**Review by agnivade — APPROVED** [review]
Thank you @kenshaw !

**Pull request merged**